helper/populate_db.py

```diff
@@ -348,7 +348,12 @@ def initialize_execution_components_table():
         InterventionComponents(intervention_component_name=Components.FIRST_AID_KIT.value,
                                intervention_component_trigger=ComponentsTriggers.FIRST_AID_KIT.value),
         InterventionComponents(intervention_component_name=Components.CLOSING_DIALOG.value,
-                               intervention_component_trigger=ComponentsTriggers.CLOSING_DIALOG.value)
+                               intervention_component_trigger=ComponentsTriggers.CLOSING_DIALOG.value),
+
+        InterventionComponents(intervention_component_name=Components.CONTINUE_UNCOMPLETED_DIALOG.value,
+                               intervention_component_trigger=None),
```

Comment:
Why do we have the trigger in ComponentsTriggers still if we do not need it?

Reply:
Unfortunately the table has the two fields. When this was designed, this case where no trigger was associated was not considered.

Reply:
@wbaccinelli I meant why we still have the trigger in the ComponentsTriggers definitions. We still define CONTINUE_UNCOMPLETED_DIALOG = "continue_uncompleted_dialog" in both ComponentsTriggers and Components.

Reply:
You are right. I changed it